stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/compose/guides/AddingCustomAttachments.kt

```diff
@@ -254,4 +259,97 @@ private object AddingCustomAttachmentsSnippet {
             )
         }
     }
+
+    /**
+     * Rename to [MessagesActivity] when adding to docs. This is so we can avoid name conflicts.
```

Comment:
I think we also add where the snippets are used here, see line 49/54 of this file.